ddtrace/contrib/starlette/patch.py

```diff
@@ -68,7 +71,15 @@ def unpatch():
 def traced_handler(wrapped, instance, args, kwargs):
     # Since handle can be called multiple times for one request, we take the path of each instance
     # Then combine them at the end to get the correct resource names
-    scope = get_argument_value(args, kwargs, 0, "scope")
+    scope = get_argument_value(args, kwargs, 0, "scope")  # type: Optional[Dict[str, Any]]
+    if not scope:
+        return wrapped(*args, **kwargs)
```

Comment:
Should we log here or even continue to hard fail? Is there a valid use-case where we'd expect to have the starlette patching applied but no tracemiddleware or request span?

Reply:
Added a log below when checking `"datadog" not in scope`, and a test case